madhumohan01-chennai.md

```diff
@@ -0,0 +1,8 @@
+# Chennai
+
+### Restaurants
+- Lots
+
+### Attractions
+- Marina
+- Shopping
```

Comment:
What kind of shopping? Can I buy a stairway to heaven? Is there food? I'm on a *See* food diet :fish: :sushi:  :wink: